plugins/filetypes/csv/write.go

```diff
@@ -0,0 +1,26 @@
+package csv
+
+import (
+	"encoding/csv"
+	"io"
+
+	"github.com/cloudquery/plugin-sdk/schema"
+)
+
+
+func WriteTableBatch(w io.Writer, table *schema.Table, resources [][]any) error {
```

Comment:
Why is this so much simpler than it was in the CSV plugin? Is it because we're now using a managed writer?

Reply:
Yes!